common/src/main/java/com/wynntils/screens/territorymanagement/widgets/GuildOverallProductionWidget.java

```diff
@@ -0,0 +1,127 @@
+        long woodUsage = holder.getOverallUsageForResource(GuildResource.WOOD);
+        long fishUsage = holder.getOverallUsageForResource(GuildResource.FISH);
+        long cropsUsage = holder.getOverallUsageForResource(GuildResource.CROPS);
+
+        List<Component> lines = new ArrayList<>();
+        lines.add(Component.literal("Guild Output").withStyle(ChatFormatting.WHITE, ChatFormatting.BOLD));
+        lines.add(Component.literal(""));
+        lines.add(Component.literal("Total resource output").withStyle(ChatFormatting.GRAY));
+        lines.add(Component.literal(""));
+
+        lines.add(Component.literal("+%d Emeralds per Hour".formatted(emeraldProduction))
+                .withStyle(ChatFormatting.GREEN));
+        lines.add(Component.literal("%s in storage".formatted(emeraldStorage)).withStyle(ChatFormatting.GREEN));
+        lines.add(
+                Component.literal("Ⓑ +%d Ore per Hour".formatted(oreProduction)).withStyle(ChatFormatting.WHITE));
+        lines.add(Component.literal("Ⓑ %s in storage".formatted(oreStorage)).withStyle(ChatFormatting.WHITE));
+        lines.add(Component.literal("Ⓒ +%d Wood per Hour".formatted(woodProduction))
+                .withStyle(ChatFormatting.GOLD));
+        lines.add(Component.literal("Ⓒ %s in storage".formatted(woodStorage)).withStyle(ChatFormatting.GOLD));
+        lines.add(Component.literal("Ⓚ +%d Fish per Hour".formatted(fishProduction))
+                .withStyle(ChatFormatting.AQUA));
+        lines.add(Component.literal("Ⓚ %s in storage".formatted(fishStorage)).withStyle(ChatFormatting.AQUA));
+        lines.add(Component.literal("Ⓙ +%d Crops per Hour".formatted(cropsProduction))
+                .withStyle(ChatFormatting.YELLOW));
+        lines.add(Component.literal("Ⓙ %s in storage".formatted(cropsStorage)).withStyle(ChatFormatting.YELLOW));
+
+        lines.add(Component.literal(""));
+
+        lines.add(Component.literal("Overall Cost (per hour):").withStyle(ChatFormatting.GRAY));
+        lines.add(Component.literal("- %d Emeralds (%.1f%%)"
+                        .formatted(emeraldUsage, (double) emeraldUsage / emeraldProduction * 100d))
+                .withStyle(ChatFormatting.GREEN));
+        lines.add(Component.literal("- Ⓑ %d Ore (%.1f%%)".formatted(oreUsage, (double) oreUsage / oreProduction * 100d))
+                .withStyle(ChatFormatting.WHITE));
+        lines.add(Component.literal(
+                        "- Ⓒ %d Wood (%.1f%%)".formatted(woodUsage, (double) woodUsage / woodProduction * 100d))
+                .withStyle(ChatFormatting.GOLD));
+        lines.add(Component.literal(
+                        "- Ⓚ %d Fish (%.1f%%)".formatted(fishUsage, (double) fishUsage / fishProduction * 100d))
+                .withStyle(ChatFormatting.AQUA));
+        lines.add(Component.literal(
+                        "- Ⓙ %d Crops (%.1f%%)".formatted(cropsUsage, (double) cropsUsage / cropsProduction * 100d))
+                .withStyle(ChatFormatting.YELLOW));
```

Comment:
Small request here: Might it be possible to show the difference instead (or when holding Ctrl)? In certain cases it might be better to know, that you have a surplus of 6k wood to use instead of knowing that you're currently using 99% of your produced wood.

Reply:
![Screenshot from 2024-05-15 11-43-17](https://github.com/Wynntils/Artemis/assets/49001742/4242c295-dd06-4290-b1d6-48115bbd3688)
@cmkohnen

Reply:
tyvm!